app/views/downloads/download_linux.html.erb

```diff
@@ -1,7 +1,15 @@
 <%- @section = "downloads" %>
 <%- @subsection = "" %>
 
-<div id="main">
+<div id="main" class="maincontent">
+  <style>
+    .maincontent pre {
+      height: 10px;
+      overflow-y: hidden;
+      padding-top: 0;
```

Comment:
I don't think this matches what `<code>` does (we still have more margin/padding on the non-top sides). It might be easier to reuse the css we use there. Or, looking at the existing css, it really does look like the intent was that one would do `<pre><code>...</code></pre>` as you had originally (I say that because there are selectors that would match `code` inside `pre`).

That involves fewer styling decisions for us to make here, plus it doesn't carry any risk of changing some other unexpected part of the page.